### PR TITLE
Rules

### LINKED ISSUE #77: As a player, I can see a splash screen explaining the rules
Rules include colour-coding, multiplayer access.

### PR DESCRIPTION
**Pull request by GiaCroock**
closes #77

### CODE REVIEW

File: src/views/multiPlayer.html [hunk 1]
```diff
@@ -4,11 +4,44 @@
...
+        <div class="modal-dialog">
+        
+        <!-- Modal content-->
+        <div class="modal-content">
+            <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal">&times;</button>
+            <h4 class="modal-title">How To Play:</h4>
+            </div>
+            <div class="modal-body">
+            <p style="color: black;">Guess a 5 letter word and enter it via the onscreen keyboard or via your physical keyboard.<br>
```
Comment (raelware) [inline src/views/multiPlayer.html:29]: Please change this to give the instructions on the two types of multiplayer setup

**Review by raelware — CHANGES_REQUESTED** [review]
Good job on making Splash screen  informative. Please see requested change

File: src/views/multiPlayer.html (hunk 1 above)
Comment (GiaCroock) [reply src/views/multiPlayer.html]: I have changed the multi-player instructions please check if you are happy with them

File: src/views/multiPlayer.html [hunk 2]
```diff
@@ -23,15 +23,27 @@
...
+        </div>
+        <div class="modal-body">
+        <h3 style="color: black;">There are two options to choose from:</h3> <nr>
+        <h4> Option 1: Two Player Game - Guessing the word of the day</h4>
+            <p style="color:black">One player needs to start the game by clicking on "CREATE 2 Player Game". This will generate a unique game ID. <br>
+        This player should then share the game ID with one other person.  <br> The second player must click on "JOIN a Game" and enter the Game ID they 
+        have recieved from the other player <br> Both players must try and guess the word of the day </p>
+        <h4> Option 2: Three Player Game -Guessing a word chosen by one of the players</h4>
+        <p style="color:black">One player needs to start the game by entering a 5-letter word of their choice and clicking on "CREATE 3 Player Game". This will generate a unique game ID. <br>
+        This player should then share the game ID with two other person.  <br> The second and third player must click on "JOIN a Game" and enter the Game ID they
```
Comment (raelware) [inline src/views/multiPlayer.html:36]: It says "two other person", please change this to two other people

File: src/views/multiPlayer.html [hunk 3]
```diff
@@ -23,15 +23,27 @@
...
-            Click Enter <br>
-            If the word you entered is valid, you will be given feedback: <br></p>  <b>
-            <p style="color:mediumseagreen";> Green: The letter is in the word of the day and in the correct position <br> </p>
-            <p style="color:DodgerBlue"> Blue: The letter is in the word of the day but is in the wrong position <br></p>
-            <p style="color:gray"> Grey: The letter is not in the word of the day <br></p></b>
+            <h3 class="modal-title">Multiplayer Information:</h3>
+        </div>
+        <div class="modal-body">
+        <h3 style="color: black;">There are two options to choose from:</h3> <nr>
+        <h4> Option 1: Two Player Game - Guessing the word of the day</h4>
```
Comment (raelware) [inline src/views/multiPlayer.html:30]: please change this from word the day to just correct word because it is not necessarily the word of the day

**Review by raelware — CHANGES_REQUESTED** [review]
Please see requested changes. When i pulled it down i saw you have instructions on the front screen as well as in the pop up. please remove the ones on the front screen

File: src/views/multiPlayer.html (hunk 2 above)
Comment (GiaCroock) [reply src/views/multiPlayer.html]: Done

**Review by raelware — APPROVED** [review]

**Pull request merged**